tests/components/geofency/test_init.py

```diff
@@ -288,16 +287,22 @@ def setup_zones(loop, hass):
 @pytest.mark.xfail(
```

Comment:
Do we still need `xfail` here?

Reply:
Yes, we'll remove it in the device tracker conversion to EntityComponent.